run-checks

```diff
@@ -135,7 +135,7 @@ if [ "$SPREAD" = 1 ]; then
     export PATH=$TMP_SPREAD:$PATH
     ( cd $TMP_SPREAD && curl -s -O https://niemeyer.s3.amazonaws.com/spread-amd64.tar.gz && tar xzvf spread-amd64.tar.gz )
 
-    spread -v linode:
+    spread -vv linode:ubuntu-core-16-64:tests/main/ubuntu-core-system-user-autoimport
```

Comment:
> Hmm, are you sure we want this change?

@zyga no way! :) but I think this is an outdated diff, it's already removed from the changeset, could you pls confirm? Thanks! :)